source/includes/_contacts.md

```diff
@@ -265,7 +265,7 @@ home_phone | The home phone for this contact. If the contact already has a home
 other_phone | Phone of unknown type for this contact. If the contact already has phone of unknown type, this overwrites it. If a contact does not already have phone of unknown type, this ADDS a new phone on top of existing phone numbers | "123-456-7890"
 
 
-## Search for Contacts
+## Search for contacts
```

Comment:
Casing update the description section on line no [254](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L254) and [255](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L255) to be consistent with [253](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L253)

`Last Name` -> `Last name`
`Company Name` -> `Company name`